src/main/java/Control/Scenario/UC_Create.java

```diff
@@ -30,6 +30,11 @@ public UC_Create(@NotNull DbProvider dbProvider) {
         try {
             db.beforeSqlExecution(true);
 
+            if (commType.getName().equals("")) {
```

Comment:
nevidim velmi do tychto CommType, ale nemali by byt povolene len urcite druhy tych CommType, cize tu netreba este podmienku okrem toho ze ci to je prazdne aj na to ci je to jeden z povolenych typov?

Reply:
No pozri sa. Síce to kodil ľudo a ja som sem pridal iba podmienku na to aby to nemohlo byť prázdne, ale ako môžeš dopredu vedieť aký je to communication type. Dnes si povieš, že je to I2C a Serial, ale za behu sa objaví podmienka, že tam bude Bluetooth alebo zajtra výjde technológia Redwing.... Nemôžeš robiť také, že povolený druh vopred inak to môžeš mať rovno v kóde a nie v db